SnowballSwiftExample/SnowballSwiftExample/Example Views/SnowballSwiftExampleNFTView.swift

```diff
@@ -34,7 +34,7 @@ struct SnowballSwiftExampleNFTView: View {
 //        )
     }
 
-    func alchemyKey(for chain: SnowballChain = .eth_mainnet) -> String {
+    func alchemyKey(for chain: Chain = .eth_mainnet) -> String {
```

Comment:
Could chain ever be cross chains? like Solona or bitcoin or whatever? Perhaps:
```swift
.eth(.main)
```

not sure if possible.

Reply:
really like this

Reply:
NOT FUN NO